src/main/java/org/spongepowered/api/effect/sound/music/MusicDiscs.java

```diff
@@ -38,20 +38,22 @@
 
     public static final Supplier<MusicDisc> CHIRP = Sponge.getRegistry().getCatalogRegistry().provideSupplier(MusicDisc.class, "chirp");
 
-    public static final Supplier<MusicDisc> ELEVEN = Sponge.getRegistry().getCatalogRegistry().provideSupplier(MusicDisc.class, "eleven");
-
     public static final Supplier<MusicDisc> FAR = Sponge.getRegistry().getCatalogRegistry().provideSupplier(MusicDisc.class, "far");
 
     public static final Supplier<MusicDisc> MALL = Sponge.getRegistry().getCatalogRegistry().provideSupplier(MusicDisc.class, "mall");
 
     public static final Supplier<MusicDisc> MELLOHI = Sponge.getRegistry().getCatalogRegistry().provideSupplier(MusicDisc.class, "mellohi");
 
+    public static final Supplier<MusicDisc> MUSIC_DISC_11 = Sponge.getRegistry().getCatalogRegistry().provideSupplier(MusicDisc.class, "11");
```

Comment:
This class is weird too - Minecraft does not expose music disc types, but we do, and so we have this weirdness. The official names of these are "11" and "13", not "eleven" and "thirteen".